README.md

```diff
@@ -19,103 +18,96 @@ You will be working with the existing code in this repository to modify a person
+[] Developer can articulately explain their strategies for extracting this logic
+[] Developer uses partials to reduce duplication of code in the view layer
 
+### User Experience
 
-### 4) ActiveRecord
+[] The application has been styled and the user can easily navigate between different portions of the application without manually entering the URL into the nav-bar or using the back button on their browser.
 
-* 4: ActiveRecord methods are used in models to supply all appropriate functionality. Methods exist on the appropriate model, and developers are not referencing other classes or `self` in models unnecessarily. Ruby enumerables are not used where ActiveRecord methods could provide the necessary functionality. The developer can explain the ActiveRecord methods they used and the relationships between ActiveRecord models.
-* 3: ActiveRecord methods are used appropriately in the database, but some Ruby enumerables may also be used. The developer uses ActiveRecord relationships appropriately, and does not call on other classes in their models.
-* 2: The developer may be calling on other classes in models (e.g. `Category.where(title: title)`), and may struggle to explain choices they made in implementing some functionality.
-* 1: Significant functionality is missing. The developer seems to not understand the methods that ActiveRecord makes available.
+### Testing
 
+[] Project has a running test suite
+[] Test suite includes robust feature tests
+[] Test suite includes tests for validations
+[] Test suite includes tests for methods that they have created on the models
 
-### 5) Views
-
-* 4: Logic has been removed from views and controllers to the full degree possible, the developer ha minimized the number of variables passed to the view, and the developer can articulately explain their strategies for extracting this logic. The developer uses partials to reduce duplication of code in the view layer.
-* 3: Limited logic that could be moved elsewhere remains in the views and/or controllers and developers are able to identify potentially opportunities to refactor.
-* 2: Significant logic leaks into the view or remains in controllers and developers show some difficulty in identifying strategies to refactor.
-* 1: Significant logic is present in the view or controllers and developers are unable to articulate potential strategies to refactor.
-
-### 6) User Experience
-
-* 4: The application has been styled and the user can easily navigate between different portions of the application without manually entering the URL into the nav-bar or using the back button on their browser.
-* 3: The application has been styled, but the user may need to use the nav-bar to enter a URL or back button to get to access some functionality.
-* 2: The application has limited styling, and it may not be clear how to navigate the application.
-* 1: The application has little to no styling and it is difficult to navigate.
+## Extensions:
 
-### 7) Testing
+### Tags
```

Comment:
@s-espinosa do we want this extension since we use it laterl? most don't seem to get to this but maybe we could add something else or have them contact us for an extension since we are giving a little more time?